jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/services/asn1/Asn1EncodedDataRouter.java

```diff
@@ -254,10 +254,13 @@ public void processEncodedTim(ServiceRequest request, JSONObject consumedObj) {
             // Case 3
             JSONObject asdObj = dataObj.getJSONObject(Asn1CommandManager.ADVISORY_SITUATION_DATA_STRING);
             try {
-              asn1CommandManager.depositToSdw(asdObj.getString(BYTES));
+               JSONObject deposit = new JSONObject();
+               deposit.put("expirationDate", request.getSdw().getExpirationDate());
+               deposit.put("encodedMsg", asdObj.getString(BYTES));
+               asn1CommandManager.depositToSdw(deposit.toString());
```

Comment:
The Asn1CommandManager is expecting a String "asdBytes" object and carrying that nomenclature down through the method. Since this is now a JSON string and no longer the asdBytes, let's update the method to reflect and avoid confusion down the road

Reply:
Updated Asn1CommandManager.depositToSdw to reflect change from bytes to object.